packages/h5p-server/test/ContentManager.test.ts

```diff
@@ -129,6 +131,33 @@ describe('ContentManager', () => {
         );
     });
 
+    it('deletes the related contentUserData', async () => {
+        const mockContentUserDataStorage = MockContentUserDataStorage();
+        await withDir(
+            async ({ path: tempDirPath }) => {
+                const contentManager = new ContentManager(
+                    new FileContentStorage(tempDirPath),
+                    mockContentUserDataStorage
+                );
+
+                const user = new User();
+                const contentId = await contentManager.createOrUpdateContent(
+                    mockupMetadata,
+                    mockupParameters,
+                    user
+                );
+
+                await expect(
+                    contentManager.deleteContent(contentId, user)
+                ).resolves.toBeUndefined();
+                await expect(
```

Comment:
It would be better if the test created content states for several users and checks if the content is really deleted for all users.

Reply:
The contentUserData is deleted based on the contentId. I will add a separate function for a specific user.